src/jenkins-enhanced.coffee

```diff
@@ -422,7 +422,7 @@ class HubotJenkinsPlugin extends HubotMessenger
   _addJobsToFoldersList: (items, server, folder) =>
     for item in items
       itemType = item._class
-      if (itemType == 'com.cloudbees.hudson.plugins.folder.Folder' or itemType == 'org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject')
+      if (itemType == 'jenkins.branch.OrganizationFolder' or itemType == 'com.cloudbees.hudson.plugins.folder.Folder' or itemType == 'org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject')
```

Comment:
The addition of the new folder type makes sense. Mind refactoring this now that we have three conditions in this expression? Perhaps an array of each of these folder strings and a method that takes the `itemType` checking if it is contained in that array.